ui/src/components/AboutDialog.tsx

```diff
@@ -0,0 +1,79 @@
+import { useDismissNavigate } from '@/logic/routing';
+import { isTalk, useCopy } from '@/logic/utils';
+import { usePike } from '@/state/kiln';
+import Dialog, { DialogContent } from './Dialog';
+import IconButton from './IconButton';
+import CheckIcon from './icons/CheckIcon';
+import CopyIcon from './icons/CopyIcon';
+
+export default function AboutDialog() {
+  const dismiss = useDismissNavigate();
+  const pike = usePike(isTalk ? 'talk' : 'groups');
+  const { didCopy: didCopyHash, doCopy: doCopyHash } = useCopy(
+    pike?.hash || ''
+  );
+  const { didCopy: didCopyShip, doCopy: doCopyShip } = useCopy(
+    pike?.sync?.ship || ''
+  );
+
+  const onOpenChange = (open: boolean) => {
+    if (!open) {
+      dismiss();
+    }
+  };
+
+  const appDescription = isTalk
+    ? `Send encrypted direct messages to one or many friends.
```

Comment:
I think if you wanted to we could pull this from the treaty instead of hardcoding

Reply:
Yeah, this is a great idea.